lib/EarlyFunctionAnalysis/EarlyFunctionAnalysis.cpp

```diff
@@ -1882,7 +1836,7 @@ bool EarlyFunctionAnalysis::runOnModule(Module &M) {
 
   // Instantiate a CFEPAnalyzer object
   using CFEPA = CFEPAnalyzer<FunctionAnalysisResults>;
```

Comment:
Can we make disappear the `CFEP` wording entirely. At this point they are no longer "candidates".

Reply:
Can `EntrypointsAnalyzer` work instead?

Reply:
Note: this is being done.